--- ddpm.py	2023-05-10 10:09:50.740215001 +0200
+++ ddpm.py	2023-05-10 10:10:20.999938136 +0200
@@ -1,3 +1,4 @@
+# %load /kaggle/working/zero123/zero123/ldm/models/diffusion/ddpm.py
 """
 wild mixture of
 https://github.com/lucidrains/denoising-diffusion-pytorch/blob/7706bdfc6f527f58d33f84b7b522e61e6e3164b3/denoising_diffusion_pytorch/denoising_diffusion_pytorch.py
@@ -17,7 +18,7 @@
 import itertools
 from tqdm import tqdm
 from torchvision.utils import make_grid
-from pytorch_lightning.utilities.distributed import rank_zero_only
+from pytorch_lightning.utilities.rank_zero import rank_zero_only
 from omegaconf import ListConfig

 from ldm.util import log_txt_as_img, exists, default, ismap, isimage, mean_flat, count_params, instantiate_from_config
